news_scraper/spiders/businessstandard.py

```diff
@@ -16,7 +16,7 @@ class BusinessStandardSpider(SitemapIndexSpider):
         "month": lambda d: d.strftime("%B").lower(),
     }
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"article/companies/", "parse"),(r"article/economy-policy/", "parse")]
```

Comment:
@ARAVINT-S-A

```
sitemap_rules = [(r"/markets/", "parse"),(r"/companies/", "parse"),(r"/economy-policy/", "parse")]
```

I think you misunderstood what I meant last time. Please change this to the above.